ddtrace/appsec/_iast/__init__.py

```diff
@@ -59,7 +60,7 @@ def ddtrace_iast_flask_patch():
     module_name = inspect.currentframe().f_back.f_globals["__name__"]
     module = sys.modules[module_name]
     try:
-        module_path, patched_ast = astpatch_module(module, remove_flask_run=True)
+        module_path, patched_ast = astpatch_module(module, remove_flask_run=False)
```

Comment:
I'm surprised we can change this `remove_flask_run` without issues

Reply:
In fact, we have to, otherwise the flask app doesn't run anymore at all.